include/amber/amber_vulkan.h

```diff
@@ -43,6 +43,9 @@ struct VulkanEngineConfig : public EngineConfig {
 
   /// The VkQueue to use.
   VkQueue queue;
+
+  /// REQUIRED. The Vulkan instance procedue loader.
```

Comment:
"procedure"

Reply:
done. I'd actually fixed it in the next CL in the chain already, heh.